packages/react-router-dom/docs/guides/quick-start.md

```diff
@@ -105,7 +151,9 @@ function Topics({ match }) {
           <Link to={`${match.url}/components`}>Components</Link>
         </li>
         <li>
-          <Link to={`${match.url}/props-v-state`}>Props v. State</Link>
+          <Link to={`${match.url}/props-v-state`}>
```

Comment:
Not sure why we need `useRouterMatch()` here, because the same could be done with
```jsx
<Link to={(match) => `${match.url}/props-v-state`}>
```

Reply:
Using `match.url` makes it a little more obvious that I'm building a relative href, IMO. But ya, using a function `to` could work too 👍